src/org/omegat/gui/dialogs/LogDialog.java

```diff
@@ -87,11 +89,7 @@ public LogDialog(java.awt.Frame parent) {
         new SwingWorker<String, Void>() {
             @Override
             protected String doInBackground() throws Exception {
-                try (FileInputStream fis = new FileInputStream(logLocation)) {
-                    return IOUtils.toString(fis, StandardCharsets.UTF_8);
-                } catch (Exception e) {
-                    return "";
-                }
+                return loadCurrentSessionLogFromFile(logLocation);
```

Comment:
Is this feature consist with SaveAs feature? the latter saves all the current file, but here show current session.